oeps/oep-0016.rst

```diff
@@ -0,0 +1,195 @@
+==========================
+OEP-16: Bootstrap Adoption
+==========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-16 </oeps/oep-0016>`            |
++---------------+-------------------------------------------+
+| Title         | Bootstrap Adoption                        |
++---------------+-------------------------------------------+
+| Last Modified | 2017-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Andy Armstrong <andya@edx.org>            |
++---------------+-------------------------------------------+
+| Arbiter       | _TBD_                                     |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Architecture                              |
++---------------+-------------------------------------------+
+| Created       | 2017-04-18                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+In order to accelerate development, and provide a more consistent user
+experience for our users, edX will adopt Bootstrap 4 to build all of its web
+applications. The Open edX community will be able to extend the platform
+using any of Bootstrap's collection of components, as well as use Bootstrap
+theming to brand their instances. An edX-supported subset of components will
+be provided that are verified to support `WCAG 2.0`_ to ensure that all new user
+interfaces conform to the `edX accessibility guidelines`_.
+
+Technology Selection
+====================
+
+The following rubric was used to assess potential technologies:
+
+* Learning curve / prior experience
+* Options for community - force them down a path or give them options
+* Frameworks that are opinionated vs. flexible
+* Ability to easily build accessible front-ends
+* Ease of design-framework integration
+* DIY framework vs. all-inclusive platform
+* Willingness to dedicate resources to development and maintenance
+* Ease of prototyping
+* Community project support
+* Compliance and security considerations
+* Availability of developers (hiring ability)
+* Ability to achieve business strategy
+
+In addition, there were a number of key goals that such a technology must
+address:
+
+* accessibility
+* internationalization and right-to-left languages
+* theming
+* performance
+* mobile-first responsive designs
+* extensibility / ease of overriding
+* modularity / scoping
+* Open edX community interest
+* size and activity of the framework's community?
```

Comment:
Is the "?" here to indicate this goal might be cut?  It seems to be by the rubric item, "Community project support."